cmd/utilities/gomobilewrappers/keyresolverwrapper.go

```diff
@@ -14,14 +14,14 @@ import (
 	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-gomobile/api"
 )
 
-// VDRKeyResolverWrapper wraps a gomobile-compatible version of a DIDResolver and translates methods calls to
+// VDRResolverWrapper wraps a gomobile-compatible version of a DIDResolver and translates methods calls to
```

Comment:
Removed `Key` since it's not actually specific to `did:key`

Reply:
Rename this file and _test file?